ports/raspberrypi/link-rp2040.ld

```diff
@@ -84,7 +84,7 @@ SECTIONS
 
         *(EXCLUDE_FILE(*libgcc.a: *libc.a:*lib_a-mem*.o *libm.a: *interp.o *divider.o *tusb_fifo.o *mem_ops_aeabi.o *usbh.o) .text*)
         /* Allow everything in usbh.o except tuh_task_event_ready because we read it from core 1. */
-        *usbh.o (.text.[_uphc]* .text.tuh_[cmvied]* .text.tuh_task_ext*)
+        *usbh.o (.text.[_uphc]* .text.tuh_[cmved]* .text.tuh_task_ext*)
```

Comment:
what's this bit about?

Reply:
This causes `tuh_inited` to be placed in RAM so that it can be called from the second core. I'd really love a linker to handle this for me. (Maybe a leave project.)

Reply:
oh I hadn't even figured out that was a regular expression or glob or something.  I was looking for sections/symbols named "cmved". thanks for the clarification!